client/src/redux/store.js

```diff
@@ -13,8 +13,8 @@ const rootReducer = combineReducers({
 const store = createStore(
   rootReducer,
   compose(
-    applyMiddleware(thunk)/* ,
-    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__() */
+    applyMiddleware(thunk),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
Pisaliśmy o tym na komunikatorze, jeżeli nie mam rozszerzenia REDUX Dev Tools w chromie, to strona leży,

```
lukasz-szewczak
Mentor
12:02
Cześć , mocno przekombinowane, n ajlepiej zawsze przeczytać dokumentację rozszerzenia gdzie z reguły wszystko jest 😉
https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup
Poza tym zgdonie z opisem https://github.com/zalmoxisus/redux-devtools-extension/issues/320#issuecomment-285643731 window.__REDUX_DEVTOOLS_EXTENSION__ nie jest przeznaczony do użycoia w compose
```

Reply:
Zgadza się, włączyłem z powodu debuggowania. Poprawię.

Reply:
Ustawiłem tym razem wersję __COMPOSE na podstawie dokumentacji
Bez instalowania dodatkowych paczek.

Reply:
Dokumentacja opisuje to trochę inaczej https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup

Reply:
Racja. nie jest to sposób przykazany